docs(custom-views): Add custom views documentation (#11390)
getsentry/sentry-docs: docs/concepts/search/saved-searches.mdx

description: "Learn more about default, recommended, and saved searches."
---

> {/* Maybe this note should be yellow/red since there is some time pressure?  */}
> <Note>
>    Saved Searches is deprecated and will be removed at the end of the year. Your saved searches will still be available in the "Add View" menu on the **Issues** page, and they can easily be converted to a Issue Views.
> </Note>
> 

Reviewer: ```suggestion
<Alert>
Saved Searches has been deprecated and will be removed at the end of the year. Your saved searches will still be available in the "Add View" menu on the **Issues** page. Any saved search can be converted to an Issue View.
</Alert>

```
